pxr/usd/lib/sdf/fileFormat.cpp

```diff
@@ -257,13 +258,39 @@ SdfFileFormat::GetFileExtension(
     return extension.empty() ? s : extension;
 }
 
+/* static */
+std::set<std::string>
+SdfFileFormat::FindAllFileFormatExtensions()
+{
+    std::set<std::string> result;
+    for (const auto& t : FindAllFileFormats()) {
+        auto factory = t.GetFactory<Sdf_FileFormatFactoryBase>();
+        auto format = factory->New();
+        for (const auto& e : format->GetFileExtensions()) {
+            result.insert(e);
+        }
+    }
+    return result;
+}
+
+/* static */
+std::set<TfType>
+SdfFileFormat::FindAllFileFormats()
+{
```

Comment:
Was this API just added as a helper for the function above? If so, I'd rather remove it since it's a pretty thin cover around pre-existing API.